addons/website_sale/views/templates.xml

```diff
@@ -669,12 +669,9 @@
                                     <t t-if="is_view_active('website_sale.search')">
                                         <t t-set="_has_all_sibligns" t-value="hasPricelistDropdown and is_view_active('website_sale.sort') and wsale_has_filters_btn"></t>
                                         <!-- Desktop Search button -->
-                                        <div
-                                            t-attf-class="o_wsale_products_header_search_form_container {{ _has_all_sibligns and 'd-none d-sm-inline-block'}} col-xl-5 me-auto {{not hasLeftColumn and 'col-xl-6'}}"
-                                            t-call="website_sale.search"
-                                        >
+                                        <div t-attf-class="o_wsale_products_header_search_form_container {{ _has_all_sibligns and 'd-none d-sm-inline-block'}} col-xl-5 me-auto {{not hasLeftColumn and 'col-xl-6'}}">
                                             <t t-if="category" t-set="placeholder">Search in <t t-out="category.name"/></t>
-                                            <t t-set="search" t-value="original_search or search"/>
+                                            <t t-call="website_sale.search" placeholder="placeholder" search="original_search or search"/>
```

Comment:
How would existing website templates in the database react to this change? Do you plan to make an upgrade script for that?

Reply:
It's backward compatible, the old semantics will be deprecated in 19.1